### DISCUSSION TITLE
npm install of private github hosted packages started failing

### DISCUSSION THREAD

**Discussion by tomaskallup** [community/community#127507] [Packages]
### Select Topic Area

Bug

### Body

Hello, since yesterday our deployments have randomly started failing on npm install.

It always shows something like this:
```
npm ERR! code FETCH_ERROR
npm ERR! errno FETCH_ERROR
npm ERR! invalid json response body at https://npm.pkg.github.com/{ourPackage} reason: Unexpected token 't', ..."***,"devD:***"typescript"... is not valid JSON
```
It fails for different packages everytime and we have no idea what to do. I tried to fetch the package manually (providing my GH token) and the JSON seemed valid, so either it returns something different to our deployment (it happens on Google Cloud) or there might be a problem on GCloud side.

Has anyone ever run into a similar issue?

**Comment by cucer** [comment]
Could you please run npm cache clean --force and then try again?

**Reply by tomaskallup** [reply]
I cannot, since the install is completely out of our hands as we deploy to Google App Engine.
The error went away on its own, so this is no longer an issue, but we still have no idea why it happened.

**Comment by Xelliese** [comment]
Try clearing the npm cache (npm cache clean --force) and double-check your token permissions. It might also be a network or firewall issue on Google Cloud’s end.
